
<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Lifeng Lin, PhD's Resume</title>
    <style>
        body {
            font-family: 'Courier New', Courier, monospace;
            margin: 20px;
            background-color: #ffffff;
            color: #333333;
        }
        .resume-header {
            text-align: center;
            margin-bottom: 30px;
        }
        .section {
            margin-top: 40px;
            padding: 10px;
            border: 1px solid #000;
        }
        .section h2 {
            font-size: 1.5em;
            border-bottom: 1px solid #000;
            padding-bottom: 5px;
        }
        .btn-download {
            margin-top: 20px;
            display: inline-block;
            text-decoration: none;
            padding: 5px 10px;
            border: 1px solid #000;
            color: #000;
            background-color: #fff;
            transition: background-color 0.3s;
        }
        .btn-download:hover {
            background-color: #f0f0f0;
        }
    </style>
</head>
<body>
    <div class="resume-header">
        <h1>Lifeng Lin, PhD</h1>
        <p>Developing Code To Enable Better Assays</p>
        <p>Phone: 224-444-0263 | Email: Bioinformatics.Scientist@gmail.com</p>
    </div>

    <div class="section" id="highlights">
        <h2>Highlights</h2>
        <ul>
            
            <li>14+ years of experience delivering robust point-of-care assays across multiple disciplines: infectious diseases, cancer early-detection and immune repertoire</li>
            
            <li>Experienced with various technology platforms: NGS, qPCR, single-cell, microarray</li>
            
            <li>Built successful bioinformatics functions to support efficient assay development</li>
            
            <li>Led a team to work cross-functionally in fast-paced projects</li>
            
        </ul>
    </div>

    <div class="section" id="experience">
        <h2>Experience</h2>
        
        <h4>Principle Scientist - Bioinformatics - Cepheid inc. (A Danaher Company)</h4>
        <p><strong>Duration:</strong> 2020.10 - present</p>
        <ul>
            
            <li>Managed Bioinformatics Engineering team, overseeing server setup, code repository organization, and workflow automation</li>
            
            <li>Built bioinformatics libraries for assay development (Python, AWS)</li>
            
            <li>Created a company-wide bioinformatics web portal for assay storage and evaluation (Flask, MySQL, Docker, nginx)</li>
            
            <li>Set up a Galaxy server for automated workflows</li>
            
            <li>Advised company leadership on bioinformatics aspects for decision-making</li>
            
            <li>Developed thermodynamic predictive models for special nucleotides</li>
            
        </ul>
        <p><strong>Reason for termination:</strong> None - still working here.</p>
        <hr>
        
        <h4>Data Scientist III - Bio-Rad Laboratories</h4>
        <p><strong>Duration:</strong> 2019.04 - 2020.10</p>
        <ul>
            
            <li>Maintained the performance of the Digital Assay Design Engine</li>
            
            <li>Led design engine updates for the Digital Assays customer portal</li>
            
            <li>Designed in-house infectious disease assays</li>
            
        </ul>
        <p><strong>Reason for termination:</strong> Resigned for a better career opportunity at Cepheid.</p>
        <hr>
        
        <h4>Panel Design Lead / System Admin - Paragon Genomics</h4>
        <p><strong>Duration:</strong> 2017.12 - 2019.04</p>
        <ul>
            
            <li>Built ParagonDesigner, an online portal to track custom design requests (WordPress + PHP + Python)</li>
            
            <li>Created a pipeline that removed bottlenecks in RNA expression assay design (Bash + Python + R)</li>
            
            <li>Set up and maintained AWS clusters for data analysis</li>
            
            <li>Administered company website, FTP, and data analysis servers</li>
            
            <li>Repackaged NGS data processing pipeline for inexperienced customers (Snakemake + Anaconda + Python)</li>
            
        </ul>
        <p><strong>Reason for termination:</strong> Relocated for family reasons.</p>
        <hr>
        
        <h4>Staff Scientist (Promoted from Scientist III) - Thermo Fisher Scientific</h4>
        <p><strong>Duration:</strong> 2015.04 - 2017.12</p>
        <ul>
            
            <li>Led the design of non-standard panels, including Immune Repertoire (patent pending) and plant barcoding</li>
            
            <li>Improved design of custom human NGS panels for FDA submissions</li>
            
            <li>Wrote design pipelines for bacterial and viral target analysis</li>
            
        </ul>
        <p><strong>Reason for termination:</strong> Joined Paragon to help a colleague with a startup.</p>
        <hr>
        
        <h4>Senior Bioinformatics Scientist - Nanosphere inc (A Luminex Company)</h4>
        <p><strong>Duration:</strong> 2010.6 - 2015.4</p>
        <ul>
            
            <li>Performed genome analysis for target pathogens</li>
            
            <li>Developed sequence databases for over 500 organisms</li>
            
            <li>Led the design of Verigene Enteric Pathogen panels (Virus, bacteria, and parasites)</li>
            
            <li>Co-designed Verigene Respiratory Pathogen assays and Blood Sepsis assays</li>
            
            <li>Built automated pipelines and portals</li>
            
        </ul>
        <p><strong>Reason for termination:</strong> Relocated to California from Illinois.</p>
        <hr>
        
        <h4>PhD Student / Research Assistant - University of Georgia, Plant Genome Mapping Lab (PGML)</h4>
        <p><strong>Duration:</strong> 2003.8 - 2010.5</p>
        <ul>
            
        </ul>
        <p><strong>Reason for termination:</strong> </p>
        <hr>
        
    </div>

    <div class="section" id="patents">
        <h2>Patents</h2>
        
        <p><strong>Compositions and Methods for Immune Repertoire Sequencing</strong></p>
        <p>Date: Filed Jan 17th, 2018 | ID: WO2020018836</p>
        <p>Inventors: Timothy Looney, Geoffrey Lowman, Lifeng Lin</p>
        <hr>
        
    </div>

    <div class="section" id="publications">
        <h2>Publications</h2>
        <p>Research Gate Profile: <a href="http://www.researchgate.net/profile/Lifeng_Lin/publications">http://www.researchgate.net/profile/Lifeng_Lin/publications</a></p>
        <h3>Journal Publications</h3>
        <ul>
            
            <li>
                <strong>Paterson, A. H., J. F. Wendell, et al.</strong> (2012). 
                “Repeated polyploidization of Gossypium genomes and the evolution of spinnable cotton fibres.” <br>
                Nature
                 492
                : 423-427.
            </li>
            
            <li>
                <strong>Lin, L., A. H. Paterson</strong> (2009). 
                “Physical composition and organization of the Gossypium genomes.” <br>
                Genetics and Genomics of Cotton
                
                .
            </li>
            
            <li>
                <strong>Lin, L., A. H. Paterson</strong> (2011). 
                “Size variation in homologous segments across divergent plant genomes.” <br>
                Mob Genet Elements
                 1
                : 92-96.
            </li>
            
            <li>
                <strong>Lin, L., G. J. Pierce, et al.</strong> (2010). 
                “A draft physical map of a D-genome cotton species (Gossypium raimondii).” <br>
                BMC Genomics
                 11
                : 395.
            </li>
            
            <li>
                <strong>Lin, L., H. Tang, et al.</strong> (2011). 
                “Comparative analysis of Gossypium and Vitis genomes indicates genome duplication specific to the Gossypium lineage.” <br>
                Genomics
                 97
                : 313-320.
            </li>
            
            <li>
                <strong>He, S., Y. Zheng, et al.</strong> (2013). 
                “Converting restriction fragment length polymorphism to single-strand conformation polymorphism markers and its application in the fine mapping of a trichome gene in cotton.” <br>
                Plant Breeding
                 132
                : 337-343.
            </li>
            
            <li>
                <strong>Paterson, A. H., J. E. Bowers, et al.</strong> (2009). 
                “Comparative genomics of grasses promises a bountiful harvest.” <br>
                Plant Physiol
                 149
                : 125-131.
            </li>
            
            <li>
                <strong>Paterson, A. H., B. K. Mashope, L. Lin</strong> (2009). 
                “The Comparative Genomics of Orphan Crops.” <br>
                ATDF Journal
                 6
                : 16-23.
            </li>
            
            <li>
                <strong>Rong, J., F. A. Feltus, L. Lin, et al.</strong> (2010). 
                “Gene copy number evolution during tetraploid cotton radiation.” <br>
                Heredity (Edinb)
                 105
                : 463-472.
            </li>
            
            <li>
                <strong>Wang, X., M. J. Torres, et al.</strong> (2011). 
                “A physical map of Brassica oleracea shows complexity of chromosomal changes following recursive paleopolyploidizations.” <br>
                BMC Genomics
                 12
                : 470.
            </li>
            
            <li>
                <strong>Ding, M., J. Chen, et al.</strong> (2014). 
                “Genome-wide investigation and transcriptome analysis of the WRKY gene family in Gossypium.” <br>
                Molecular Genetics and Genomics
                
                .
            </li>
            
            <li>
                <strong>Ding, M, Y. Jiang, et al.</strong> (2013). 
                “Gene expression profile analysis of Ligon lintless (Li1) mutant reveals important insights.” <br>
                Unknown
                
                .
            </li>
            
        </ul>
        <h3>Conference Appearances</h3>
        <ul>
            
            <li>
                <strong>Lucie Li, Lily Liu, Jacey Zhang, Jeffery Liu, Lifeng Lin, Guoying Liu, Zhitong Liu</strong> (2018). 
                “Detecting low allele frequency somatic mutations using targeted amplicon sequencing and a novel double strand molecular barcoding scheme.” <br>
                AACR Annual Meeting 2018
                
                
                .
            </li>
            
            <li>
                <strong>Timothy Looney, Geoffrey Lowman, Elizabeth Linch, Denise Topacio, Lauren Miller, Lifeng Lin, Aleksandr Pankov, Janice Au-Young, Mark Andersen, Ann Mongan, Fiona Hyland</strong> (2017). 
                “Tracking the interplay between circulating and tumor-infiltrating T cells using AmpliSeq-based Ion Torrent TCRβ immune repertoire sequencing.” <br>
                
                , Journal of Clinical Oncology
                 35
                : e14518.
            </li>
            
            <li>
                <strong>Ann Mongan, Geoffrey Lowman, Yuan-Chieh Ku, Timothy Looney, Lifeng Lin, Xinzhan Peng, Fiona Hyland</strong> (2017). 
                “Using NGS to characterize 40 NSCLC tumor with gene expression and tumor infiltrating T cell repertoire profiling from FFPE and matched Fresh Frozen samples.” <br>
                Proceedings of the American Association for Cancer Research Annual Meeting 2017
                
                
                .
            </li>
            
            <li>
                <strong>Denise S. Topacio-Hall, Tim Looney, Yongming Sun, Lauren Miller, Elizabeth Linch, Geoffrey Lowman, Lifeng Lin, Mark Andersen, Fiona Hyland, Ann Mongan</strong> (2017). 
                “Sequencing the human TCRβ repertoire on the Ion S5™ System.” <br>
                
                , Cancer Research
                 77
                .
            </li>
            
            <li>
                <strong>S. Marla, M. Dado, C. Gerstein, et al.</strong> (2013). 
                “Development of Nanosphere’s Verigene BC-GN Test for rapid detection of Gram-negative bacteria and resistance determinants directly from positive blood cultures.” <br>
                23rd European Congress of Clinical Microbiology and Infectious Diseases
                
                
                .
            </li>
            
            <li>
                <strong>Lin, L., Rong, J., Paterson, A.H.</strong> (2009). 
                “Progress toward cloning a gene involved in cotton fiber development (Ligon lintless-2, Li2) with the assistance of genetically anchored BAC libraries.” <br>
                Large Insert DNA Libraries and Their Applications Workshop at Plant &amp; Animal Genome XVII
                
                
                .
            </li>
            
            <li>
                <strong>Lifeng Lin, Gary Pierce, John E. Bowers, et al.</strong> (2010). 
                “A Draft Physical Map of a D-genome Cotton Species (G. ramondii).” <br>
                Plant &amp; Animal Genome XVIII
                
                
                .
            </li>
            
            <li>
                <strong>Tang, H., Lin, L., Bowers, J.E., Rong, J., Paterson A.H.</strong> (2006). 
                “Large Scale EST Resource Compilation and Applications for Phylogenomics of Major Crop Plants.” <br>
                Annual SMBE Meeting 2006
                
                
                .
            </li>
            
            <li>
                <strong>Lin, L., Pierce, G., Bowers, J.E., et al.</strong> (2007). 
                “Progress toward a fingerprinted physical map of D genome cotton G. raimondii.” <br>
                Plant &amp; Animal Genome XV
                
                
                .
            </li>
            
        </ul>
    </div>

    <div class="section" id="profiles">
        <h2>Public Profiles</h2>
        <ul>
            
            <li><a href="https://www.linkedin.com/in/lifeng-phil-lin-23a20111" target="_blank">LinkedIn</a></li>
            
            <li><a href="https://github.com/linlifeng/" target="_blank">GitHub</a></li>
            
            <li><a href="http://www.researchgate.net/profile/Lifeng_Lin/publications" target="_blank">ResearchGate</a></li>
            
        </ul>
    </div>

    <div class="section" id="education">
        <h2>Education</h2>
        
        <p>PhD - University of Georgia (2010)</p>
        <hr>
        
        <p>B.S. - Fudan University (2003)</p>
        <hr>
        
    </div>

    <div class="section" id="certifications">
        <h2>Certifications</h2>
        <ul>
            
            <li>
                <strong>HarvardX: PH525.2x Linear Models and Matrix Algebra</strong> (2016)
            </li>
            
            <li>
                <strong>HKUSTx: COMP102.1x Introduction to Java Programming</strong> (2016)
            </li>
            
            <li>
                <strong>The Data Scientist&#39;s Toolbox</strong> (2015)
            </li>
            
            <li>
                <strong>R Programming</strong> (2015)
            </li>
            
            <li>
                <strong>Design Control Concepts and Implementation Workshop</strong> (2014)
            </li>
            
            <li>
                <strong>MITx: 6.00.2x Introduction to Computational Thinking and Data Science</strong> (2014)
            </li>
            
            <li>
                <strong>MITx: 6.00.1x Introduction to Computer Science and Programming Using Python</strong> (2014)
            </li>
            
            <li>
                <strong>UC-Davis Bioinformatics Training Program</strong> (2014)
            </li>
            
            <li>
                <strong>Excellence in Documentation of Teaching Proficiency</strong> (2010)
            </li>
            
        </ul>
    </div>

    
    <div class="btn-download">
        <a href="https://oligodesign.pythonanywhere.com/export/pdf">Download as PDF</a>
    </div>
    
</body>
</html>
